### DISCUSSION TITLE
Homer 10 Support: Is source code available?

### DISCUSSION THREAD

**Discussion by hxtrieu** [sipcapture/homer#666] [Q&A] [answered]
Hi everyone,

Currently, I only see support for Homer 10 in Docker. Is there a source code available for Homer 10 that I can customize, like for Homer 7 ?

**Comment by lmangani** [comment, marked as answer]
Hello @hxtrieu we have examples in docker but absolutely no dependency on docker. You can take every single element from the compose example in https://github.com/sipcapture/homer-docker and run it locally. The backend components are the same (heplify-server) and everything is opensource as always, but there's no dedicated UI, only Grafana and our experimental grafana-flow plugin. That's all :)
